### PR TITLE
Connect the gamepage winnings to myPoke

### LINKED ISSUE #86: Connect pokemon randomizer to slot machine
**Is your feature request related to a problem? Please describe.**
The randomizer function will be triggered every time the reel is spun therefore giving the user a random pokemon.

### PR DESCRIPTION
**Pull request by gonzalespaulb**
## Changes
1. Reel animation in sync with randomizer function
2. Game winnings are connected to myPoke

## Purpose
Only to displays pokemons that have been won via slot machine

## Approach
Connected redux state as a reference for the randomizer function but only used local state for the winnings so that the game winnings grid resets after the component has been unmounted.

Closes #86

### CODE REVIEW

File: pokeapp/src/components/GamePage.js [hunk 1]
```diff
@@ -1,25 +1,27 @@
...
-    const gamepokemon = useSelector((state) =>
-    state.pokemon.allPokemon.filter((pokemon) => pokemon.quantity > 0)
-  );
+// ------------------------------------------------------------------------------------------------RANDOMIZER LOGIC START
+
+  const [wonPokemon, setWonPokemon] = useState([]);
 
-    const gamePagePokemons = () => {
-        return gamepokemon.map((pokemon) => {
+  const dispatch = useDispatch();
```
Comment (Lap343) [inline pokeapp/src/components/GamePage.js:19]: some useDispatch nice need to look into this

File: pokeapp/src/components/GamePage.js [hunk 2]
```diff
@@ -34,8 +36,24 @@ const GamePage = ({makeUpperCase, setIsPokeList}) => {
         });
       };
 
+  const winAPokemon = (pokeList) => {
+    if(!reel1) {
+      let randomPokemon = randomPoke(pokeList);
+      setWonPokemon([...wonPokemon, randomPokemon])
+      dispatch(winPokemon(randomPokemon))
+    } else {
+      return;
```
Comment (Lap343) [inline pokeapp/src/components/GamePage.js:45]: cant wait to play this game

**Review by Lap343 — APPROVED** [review]
Nice use of the useDispatch code looks good to me

File: pokeapp/src/components/GamePage.js (hunk 2 above)
Comment (JorgeLVilla) [reply pokeapp/src/components/GamePage.js:45]: I second that 100%

**Review by JorgeLVilla — APPROVED** [review]
Looks great Paul! Can't wait to see it :)

**Pull request merged**